src/webgpu/shader/execution/shader_io/fragment_builtins.spec.ts

```diff
@@ -1726,31 +1728,57 @@ function checkSubgroupInvocationIdConsistency(
   const uintsPerRow = bytesPerRow / 4;
   const uintsPerTexel = (bytesPerBlock ?? 1) / blockWidth / blockHeight / 4;
 
-  const mappings = new Map<number, bigint>();
+  const invocationIdBitmapOfSubgroups = new Map<number, bigint>();
+  const ballotSizeRecordOfSubgroups = new Map<
+    number,
+    { ballotSize: number; row: number; col: number }
+  >();
   for (let row = 0; row < height; row++) {
     for (let col = 0; col < width; col++) {
       const offset = uintsPerRow * row + col * uintsPerTexel;
       const id = data[offset];
-      const size = data[offset + 1];
-      const repId = data[offset + 2];
+      const sgSize = data[offset + 1];
+      const ballotSize = data[offset + 2];
+      const repId = data[offset + 3];
```

Comment:
This seems incorrect for the shader that is run. The fragment shader returns `vec4u(id, ballotSize, repId, 0)`. So:
* offset + 0: id (this matches)
* offset + 1: ballotSize (this doesn't match)
* offset + 2: repId (this doesn't match)

Because this grabs `repId` from `offset + 3` it seems that all values will be zero, which will hit the continue statement and not perform the other checks. Is there a missing shader modification here?

Reply:
Oops... Sorry the shader change is missed. Fixed it and the test pass on Intel UHD630.